### ISSUE TITLE
Need standard for branch naming

### ISSUE THREAD

**Issue by berniedurfee** [Lambda-School-Labs/labs-engineering-standards#49]
At least a standard that all branch names need to be lowercase and include only hyphens, characters and numbers.

**Comment by LeTanque** [comment]
I agree with this idea.

To add to the naming convention, feature branches are very effective. This is my ever evolving currently defined workflow:
- Each branch is checked out from the most recent master.
- Branch names consist of 4 digit month/day and then the feature that branch addressed (examples below).
- After a branch has been merged, I delete it. Otherwise I get a long list of abandoned branches and reusing branches introduces problems.
- All branch names have no spaces, include a hyphen between the date and feature.
- Features are camel case instead of all lowercase (but I've used all lowercase and that's adequate in my opinion.

Some example branch names from a CS sprint:
`0205-mapStyling`
or 
`0206-characterControls`
